content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -147,6 +149,8 @@ Run the following commands to set up and deploy Kubeflow.
     
 Choose either **single user** or **multi-tenant** section based on your usage.
 
+If you make any mistake that creates conflicts on your Kubeflow deployment, you can [uninstall Kubeflow](#uninstall-kubeflow) and start over again.
```

Comment:
I think the signal would be some kind of issues experienced by the user - the signal that they potentially made mistakes. Therefore:

```suggestion
If you're experiencing issues during the installation because of conflicts on your Kubeflow deployment, you can [uninstall Kubeflow](#uninstall-kubeflow) and install it again.
```

WDYT?